ibm/mas_devops/roles/suite_app_config/vars/defaultspecs/predict.yml

```diff
@@ -5,6 +5,9 @@ mas_appws_spec:
     jdbc: "{{ mas_appws_bindings_jdbc | default( 'system' , true) }}"
     watsonstudio: system
   settings:
+    mat:
+      datamartid: "{{ cpd_wos_datamart_id }}"
+      install: true
```

Comment:
Do we want this to always be true like this, for all versions of predict, even when cpd_wos_datamart_id == ""?

Reply:
Hi @durera
for install MAT we have 2 option: true/false
```
if MAT install true - then we need to set CPD_INSTALL_PLATFORM true/false
if MAT install false - then end

if CPD_INSTALL_PLATFORM true - then we need to set CPD_INSTALL_OPENSCALE true
if CPD_INSTALL_OPENSCALE true - then we need to set CPD_WOS_DATAMART_ID
if CPD_WOS_DATAMART_ID true - then end

if CPD_INSTALL_PLATFORM false - then we need provide following variables:
- CPD_WML_URL
- CPD_ADMIN_USERNAME
- CPD_ADMIN_PASSWORD
- CPD_WOS_DATAMART_ID
then end
```

Reply:
So, MAT is related to whether OpenScale is being used or not?  This is an optional dependency for Predict in the install logic so when it's not installed shouldn't this all be gated something like:

```
{% if cpd_wos_datamart_id is defined and cpd_wos_datamart_id  != "" %}
    mat:
      datamartid: "{{ cpd_wos_datamart_id }}"
{% endif %}
```

Otherwise we will set this to true for every install, whether OpenScale is set up or not.